cookbooks/aws-parallelcluster-environment/resources/ec2_udev_rules/partial/_common_udev_configuration.rb

```diff
@@ -67,8 +58,7 @@
 end
 
 action :start_ec2blk do
-  service "ec2blkdev" do
-    supports restart: true
-    action %i(enable start)
+  execute "Refresh UdevAdmin" do
+    command "udevadm trigger --action=change --subsystem-match=block"
```

Comment:
May you please add a brief comment about why this is required?

Reply:
I am not 100% sure that this is required. This was done in the old code. Therefore I kept here to stay on the safe side

Reply:
Let's validate in a follow up activity if it is required or not. Better to not have not required steps